CK3toEU4/Source/Mappers/ReligionDefinitionMapper/ReligionDefinitionMapping.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ReligionDefinitionMapping.h"
+#include "ParserHelpers.h"
+
+mappers::ReligionDefinitionMapping::ReligionDefinitionMapping(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void mappers::ReligionDefinitionMapping::registerKeys()
+{
+	registerKeyword("country", [this](const std::string& unused, std::istream& theStream) {		
+		country = commonItems::singleItem(unused, theStream);
+		if (country.size() > 1)
```

Comment:
Is this removing quotes? If so, you can instead use commonItems::remQuotes in "StringUtils.cpp"

Reply:
No, it's removing singleItem opening and closing braces. singleItem will eat equal sign on its own.
Overall singleItem is not really a carbon-copy function as it will internally explode if the contents of the block it's quoting are broken, so it's applicalbe here to both transfer raw text and check its pdx syntax at the same time.